assets/panel/debugger.properties

```diff
@@ -47,6 +47,10 @@ expandPanes=Expand panes
 # button when the debugger is in a running state.
 pauseButtonTooltip=Pause %S
 
+# LOCALIZATION NOTE (pauseButton): The label that is displayed for the dropdown pause
+# list item when the debugger is in a running state, excluding the shortcut key.
```

Comment:
Can you explain what "excluding the shortcut key" means in all these localization comments?